src/tiles/mediatype-icon.ts

```diff
@@ -68,7 +68,7 @@ export class MediatypeIcon extends LitElement {
       }
 
       .fill-color {
-        fill: var(--iconCustomFillColor, var(--iconFillColor, '#000000'));
+        fill: var(--iconFillColor, '#000000');
```

Comment:
Blocker: Now that the collection icon has a fill color, we can get rid of `private get collectionColor()` and `--iconCustomFillColor` in tile-list.ts and tile-list-compact.ts

Need to handle this in the same branch.

Reply:
updated now...